kythe/typescript/testdata/type_alias.ts

```diff
@@ -1,13 +1,17 @@
 export {}
 
-//- @A defines/binding _A
+//- @A defines/binding A
 interface A {
   a: number;
 }
 
-// TODO: see TODO in visitTypeAlias.
-// B aliases A
-
 //- @B defines/binding B
 //- B.node/kind talias
+//- B aliases A
 type B = A;
+
+//- @T1 defines/binding T1
+type T1 = Array<string>;
+//- @T2 defines/binding T2
+//- !{ T2 aliases T1 }
+type T2 = Array<string>;
```

Comment:
Sorry, github ate my comment.  Here, I think we wanna verify that if given

```
type T1 = Array<string>;
type T2 = Array<number>;
```

T1 and T2 don't alias the same thing.  I think they might -- the above code makes them both alias Array because it ignores the generic in the vname generation.

Reply:
Yeah, they do point to the same thing. Let's see if we can change the VName at all.

Reply:
Nevermind, I see. We just don't emit a reference at all.